openshift/e2e-tests-openshift.sh

```diff
@@ -305,10 +306,10 @@ failed=0
 
 (( !failed )) && build_knative_client || failed=1
 
-(( !failed )) && install_servicemesh || failed=1
-
 (( !failed )) && install_knative_serving || failed=1
 
+(( !failed )) && install_servicemesh || failed=1
+
```

Comment:
Have not dive into the script yet,  but not sure why the servicemesh should be installed after knative when knative would need the service mesh ?